16-draft.json

```diff
@@ -258,6 +258,13 @@
                 "description": "Email address which can be base64 encoded.",
                 "type": "string"
               },
+              "telegram": {
+                "description": "Telegram channel link",
+                "type": "string",
+                "examples": [
+                  "https://t.me/example-space"
+                ]
+              },
```

Comment:
Is it currently in the "keymasters" key? Could make sense, but would need a direct Telegram link to a *person* not a space channel.

But I think you wanted to add it to the parent `contacts` key.

Reply:
Yes, I added it to the wrong section. Should be fixed now.